experimental/examples/bubblebench/tree/src/cursor/schema.ts

```diff
@@ -0,0 +1,66 @@
+/*!
```

Comment:
experimental/examples/bubblebench/tree/src/editable-tree/schema.ts should be left in a non editable-tree specific location, and the cursor version should reuse that instead of making a separate (slightly different) schema.

Reply:
If you want to merge this change, as is, then do the schema unification in a followup, that would be ok, buyt note that in the PR description

Reply:
Yeah the intent was to leave it as-is for now and to follow up with schema unification. I just added a blurb to the end of the description to make that clear.